Library/Homebrew/rubocops/service.rb

```diff
@@ -16,12 +16,23 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
           service_node = find_block(body_node, :service)
           return if service_node.blank?
 
-          # This check ensures that `bin` is not referenced because
-          # `opt_bin` is more portable and works with the API.
-          find_every_method_call_by_name(service_node, :bin).each do |bin_node|
-            offending_node(bin_node)
-            problem "Use `opt_bin` instead of `bin` in service blocks." do |corrector|
-              corrector.replace(bin_node.source_range, "opt_bin")
+          # This check ensures that cellar paths like `bin` are not referenced
+          # because their `opt_` variants are more portable and work with the
+          # API.
+          corrections = {
+            bin:      :opt_bin,
+            libexec:  :opt_libexec,
+            pkgshare: :opt_pkgshare,
+            prefix:   :opt_prefix,
+            sbin:     :opt_sbin,
+            share:    :opt_share,
+          }
```

Comment:
Nit: Would this make more sense as a constant?

Reply:
I think so! Updated.